src/inputwidgets/mainwindowinputqso.cpp

```diff
@@ -639,6 +640,18 @@ void MainWindowInputQSO::slotFreqRXChanged(double _f)
     }
     */
     //qDebug() << "MainWindow::slotFreqRXChanged: END" << endl;
+    setSplitCheckBox();
 }
 
+void MainWindowInputQSO::setSplitCheckBox()
+{
+    if (txFreqSpinBox->value() == rxFreqSpinBox->value())
```

Comment:
This comparison should be like:
If (util->isSameFreq(txFreqSpinBox->value(), rxFreqSpinBox->value()))